lib/factories/wallet-factory.js

```diff
@@ -82,6 +82,19 @@ function WalletFactory (lemonway) {
       });
   };
 
+  this.getBatch =
+  this.getWalletDetailsBatch = function (ip, wallets, _opts) {
+    const opts = _opts || {};
+
+    opts.wallets = wallets;
+
+    return lemonway._client.getWalletDetailsBatch(ip, opts)
+      .bind(this)
+      .then(function (data) {
+        return _.map(data, (wallet) => new this.Wallet(_.get(wallet, 'WALLET')));
```

Comment:
As discussed on slack, let's correctly map `E` to `LemonwayError` and return an object `{wallets: [], errors: []}`